src/cli.ts

```diff
@@ -46,6 +46,11 @@ class Docgen extends Command {
       description: 'path to an alternative solc module',
     }),
 
+    'solc-settings': flags.string({
```

Comment:
I think the build is failing because it's failing to infer the correct type of the `solc-settings` flag. You're defining it as a `string` yet the `parse` function returns an object.

Please use `flags.build` like the `exclude` flag above.

Reply:
Done, thanks!